crates/fstab-generate/src/block.rs

```diff
@@ -23,6 +23,7 @@ impl<'a> BlockInfo<'a> {
         target: Option<&Path>,
         options: &'a str,
     ) -> Self {
+        let pass = if uid.id == "ROOT" { true } else { false };
```

Comment:
The `ROOT` UUID is a bogus value used by the unit tests. Use the target path instead of the UUID to check if it is root, like so:

```rs
let pass = target == Some(Path::new("/"));

Reply:
I'm wondering if there is any reason to avoid applying it for all partitions.

Reply:
I did a quick test building with this change, and I can confirm it is now working.

Reply:
@mmstick Ah, right. That makes sense. I was lazily relying on the tests. I have updated the block_info_root test as well with your suggestion

Let me know if this looks better